articles/logs/query-syntax.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Logs Query Syntax
+description: Learn about Auth0's log search query string syntax.
+toc: true
+topics:
+  - logs
+  - log-management
+  - search
+  - query-syntax
+contentType:
+  - reference
+useCase:
+  - manage-logs
+---
+
+# Logs Query Syntax
+
+The [Management API logs endpoint](/api/v2#!/Logs/get_logs) allows to retrieve logs either by checkpoint (last retrieved log id) or by search criteria. We highly recommend to use the checkpoint approach to export logs to the external system of your choice and perform any search or analysis there, as logs stored in our system are subject to [the retention period](/logs#how-long-is-log-file-data-available). You can use any of the [Export Auth0 logs to an external service](/extensions#export-auth0-logs-to-an-external-service) extensions to export the logs to the system of your choice (like Sumo Logic, Splunk or Loggly).
```

Comment:
"...allows **you** to retrieve..."

Reply:
the section was removed and added similar paragraph in /logs article